packages/cli/src/wrangler.ts

```diff
@@ -9,18 +9,30 @@ import { readSession } from "./session.ts";
 
 const envFile = ".dev.vars";
 
-export async function getCurrentEnvVars(projectRoot: string) {
-  const devVarsFile = await Deno.readTextFile(
-    join(projectRoot, envFile),
-  ).catch(() => "");
-  return devVarsFile.split("\n").reduce((acc, line) => {
-    if (!line) {
+export async function getCurrentEnvVars(projectRoot: string): Promise<{
```

Comment:
Wdyt about using a lib that only handles parse env file. Example
https://www.npmjs.com/package/dotenv